cluster/aws.go

```diff
@@ -486,13 +484,24 @@ func GetKubicornProfile(cs *model.ClusterModel) *kcluster.Cluster {
 func (c *AWSCluster) GetStatus() (*components.GetClusterStatusResponse, error) {
 	log.Info("Start get cluster status (amazon)")
 
+	nodePools := make(map[string]*components.NodePoolStatus)
+	for _, np := range c.modelCluster.Amazon.NodePools {
```

Comment:
In case of azure nil np was checked, is that required here also?

Reply:
yes, i fixed it